.github/workflows/ui-test.yml

```diff
@@ -0,0 +1,33 @@
+name: Automated Tests - PR
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+        
+      - name: Run UI tests
+        uses: GabrielBB/xvfb-action@v1.5
+        with:
+          run: npm run ui-test
+
+      - name: Upload screenshots
+        uses: actions/upload-artifact@v2
+        if: failure()
+        with: 
+          name: screenshots
+          path: ~/work/fabric8-analytics-vscode-extension/screenshots/*
```

Comment:
can you point me to this working example of this maaybe

Reply:
here https://github.com/immanishbainsla/fabric8-analytics-vscode-extension/actions/runs/892176851
but only I can download this.

Reply:
as I am author of this repo